bolt/include/bolt/Core/BinaryFunction.h

```diff
@@ -143,7 +143,7 @@ class BinaryFunction {
   enum {
     PF_NONE = 0,     /// No profile.
     PF_LBR = 1,      /// Profile is based on last branch records.
-    PF_SAMPLE = 2,   /// Non-LBR sample-based profile.
+    PF_IP = 2,       /// Non-LBR sample-based profile.
```

Comment:
Why not `PF_BASIC`?

Reply:
I thought of a concise counterpart to PF_LBR. PF_BASIC would be best matched by PF_BRANCH imo. But I don't feel strongly about this.